### ISSUE TITLE
Unable to deploy jar (NoSuchMethodError)

### ISSUE THREAD

**Issue by ghost** [takezoe/gitbucket-maven-repository-plugin#5]
### Error:
```
java.lang.NoSuchMethodError: gitbucket.core.util.SyntaxSugars$.using(Ljava/lang/Object;Lscala/Function1;Lscala/Function1;)Ljava/lang/Object;
	at io.github.gitbucket.mavenrepository.controller.MavenRepositoryController.$anonfun$new$20(MavenRepositoryController.scala:161)
	at scala.util.Either.map(Either.scala:350)
	at io.github.gitbucket.mavenrepository.controller.MavenRepositoryController.$anonfun$new$19(MavenRepositoryController.scala:149)
	at scala.util.Either.flatMap(Either.scala:338)
	at io.github.gitbucket.mavenrepository.controller.MavenRepositoryController.$anonfun$new$16(MavenRepositoryController.scala:145)
	at scala.util.Either.flatMap(Either.scala:338)
	at io.github.gitbucket.mavenrepository.controller.MavenRepositoryController.$anonfun$new$14(MavenRepositoryController.scala:143)
```

**Comment by bharatsingh-tc** [comment]
Getting the same error while using SBT setup and WebDAV to publish jar, anyone found the solution yet ?

**Comment by bharatsingh-tc** [comment]
Manually changing the version in `build.sbt` within the plugin's source code from `4.23.0` to `4.23.1` allowed me to publish jars without any error.

Changing the version of `GitBucket` being used breaks the plugin for previous versions, so we would need a new version of the plugin `v1.3.1` for the new version for `GitBucket v4.23.1`

**Comment by takezoe** [comment]
Fixed in 9589ee168a1f8c5eb79d6183ac638faba36e1b8d
